common-lib/src/cmr/common/memorydb/connection.clj

```diff
@@ -0,0 +1,88 @@
+(ns cmr.common.memorydb.connection
```

Comment:
I think I'm on board with all the changes except renaming memory-db to memorydb. Can we put that back?

Reply:
Sure.

Reply:
Done.